virttest/qemu_qtree.py

```diff
@@ -471,7 +471,10 @@ def check_guests_proc_scsi(self, info):
         scsis = set()
         for scsi in _scsis:
             # Ignore IDE disks
-            if scsi[5] != 'CD-ROM':
+            # In "/proc/scsi/scsi",device info for ide/scsi almost the same,
+            # except ide.vendor="ATA" and scsi.vendor="QEMU", so use the field
+            # to distiguish scsi from ide.
+            if scsi[5] != 'CD-ROM' and scsi[4] == 'QEMU':
```

Comment:
From L457 and L458 we can see the list also contains some usb2 devices, so are you sure that the `scsi[4] == 'QEMU'` check won't filter out them?